<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_clientsSearch</class>
 <widget class="QWidget" name="form_clientsSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поиск клиентов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_clientName">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Клиент: Ф. И. О.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_searchAll">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Все</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_seach">
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_clients">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Отчество</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_clientName</tabstop>
  <tabstop>pushButton_searchAll</tabstop>
  <tabstop>pushButton_seach</tabstop>
  <tabstop>tableWidget_clients</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
